nucypher/datastore/datastore.py

```diff
@@ -183,7 +183,8 @@ def attach_kfrag_to_saved_arrangement(self, alice, id_as_hex, kfrag, session=Non
         if policy_arrangement.alice_verifying_key.key_data != alice.stamp:
             raise alice.SuspiciousActivity
 
-        policy_arrangement.kfrag = bytes(kfrag)
```

Comment:
this is consistently and reproducibly not actually happening in live contexts.

Reply:
This appears to be the syntax of FlaskSQLAlchemy update, however, FlaskSQLAlchemy is not implemented here :thinking:

Reply:
related to #2088